### PR TITLE
TRITON-2185 VMAPI should consistently set VM tags to {} when empty

### PR DESCRIPTION
**Pull request by kusor**
This is related to the multiple changefeed messages related to VM tags for machines w/o any tags: https://github.com/joyent/sdc-cloudapi/pull/68#discussion_r531736913

### CODE REVIEW

File: lib/common/vm-common.js [hunk 1]
```diff
@@ -195,6 +195,10 @@ function translateVm(obj, fullObject) {
     } else {
         vm.image_uuid = obj.image_uuid;
     }
+    // Consistently set VM tags to {} when empty:
+    if (vm.tags === undefined) {
```
Comment (twhiteman) [inline lib/common/vm-common.js:199]: Might be better to use `if (!vm.tags) {` here - to catch the case where it's null as well.

**Review by twhiteman — CHANGES_REQUESTED** [review]
Also, please bump the package.json version.

File: lib/common/vm-common.js (hunk 1 above)
Comment (kusor) [reply lib/common/vm-common.js]: Sure, done. The reason to do not add it initially was to keep the same approach than the rest of that file ... but for the case of tags has sense to represent "null" as {} too.

**Comment by twhiteman** [comment]
Looks good, but I think you missed this comment: "Also, please bump the package.json version."

**Comment by kusor** [comment]
> Looks good, but I think you missed this comment: "Also, please bump the package.json version."

Definitely :-)

**Review by twhiteman — APPROVED** [review]

**Pull request merged**